dotCMS/src/main/java/com/dotcms/rest/api/v1/site/HostForm.java

```diff
@@ -0,0 +1,125 @@
+package com.dotcms.rest.api.v1.site;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class HostForm {
```

Comment:
doc

Reply:
We need to stop using the word "Host", and use "Site" instead